src/Git/Repository.php

```diff
@@ -51,23 +51,31 @@ public function getCommitsLogPatch($file)
             . "<commiter_date>%ct</commiter_date>"
             . "<message><![CDATA[%s]]></message>"
             . "<body><![CDATA[%b]]></body>"
-            . "</item>\" -- \"$file\""
+            . "</item>\" --max-count=".($max_count+1)." $revisions -- \"$file\""
```

Comment:
I'm not sure if PSR2 dictates if there should be a space before and after the dot in string concatenation. In my opinion it would improve readability.